### PR TITLE
Add isUnique.yaml and lonelyNumber.yaml problems

### PR DESCRIPTION
**Pull request by nicholashusin**

### CODE REVIEW

File: problems/isUnique.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+id: 6
+title: IsUnique
+text: |
+  Write a function called isUnique that takes in an array of integers.
+
+  Return true if all the elements of the array are unique.
+  If not, return false.
```
Comment (lalaAdasia00) [inline problems/isUnique.yaml:7]: Could you insert an example
```suggestion
  isUnique([1, 2, 3]) = true
  isUnique([1, 1, 5, 5]) = false
```

File: problems/lonelyNumber.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+title: LonelyNumber
+text: |
+  Write a function called findLonely that takes in an array of integer.
+  The given array of integer will always have exactly one unique number, called the lonely number.
+
+  Other numbers than the lonely number will always exist in multiples of two.
+  An example of such an array is [2, 2, 1, 5, 5, 5, 5] where 1 is the lonely number
+
+  findLonely should return the value of the lonely number that it finds.
+
```
Comment (lalaAdasia00) [inline problems/lonelyNumber.yaml:11]: Same here, could you insert an example input and output.

findLonely([1, 1, 2, 2, 7, 8, 8]) = 1

Even though the description is good. Some people also prefer to see something like this.

File: problems/lonelyNumber.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+id: 7
+title: LonelyNumber
+text: |
+  Write a function called findLonely that takes in an array of integer.
```
Comment (ifeomi) [inline problems/lonelyNumber.yaml]:
```suggestion
  Write a function called findLonely that takes in an array of integers.
```

File: problems/lonelyNumber.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  The given array of integer will always have exactly one unique number, called the lonely number.
```
Comment (ifeomi) [inline problems/lonelyNumber.yaml]:
```suggestion
  The given array of integers will always have exactly one unique number, called the lonely number.
```

File: problems/lonelyNumber.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+solution: |
+  function findLonely(arr) {
+    var lonelyNumber = arr[0];
+
+    for (var i = 1; i < arr.length; ++i) {
+      lonelyNumber = lonelyNumber ^ arr[i];
+    }
+
+    return lonelyNumber;
+  };
```
Comment (ifeomi) [inline problems/lonelyNumber.yaml]:
```suggestion
  }
```

File: problems/lonelyNumber.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  - assert(findLonely([1, 2, 2, 3, 3]) === 1);
+  - assert(findLonely([3, 3, 2, 3, 3]) === 2);
+  - assert(findLonely([1, 2, 2, 3, 3, 3, 3]) === 1);
+  - assert(findLonely([3, 3, 2, 3, 3, 3, 3]) === 2);
+solution: |
+  function findLonely(arr) {
+    var lonelyNumber = arr[0];
+
+    for (var i = 1; i < arr.length; ++i) {
+      lonelyNumber = lonelyNumber ^ arr[i];
```
Comment (ifeomi) [inline problems/lonelyNumber.yaml:33]: cool solution :)

**Review by ifeomi — CHANGES_REQUESTED** [review]

File: problems/isUnique.yaml (hunk 1 above)
Comment (nicholashusin) [reply problems/isUnique.yaml:7]: Thanks for the feedback, added some examples!

File: problems/lonelyNumber.yaml (hunk 2 above)
Comment (nicholashusin) [reply problems/lonelyNumber.yaml:11]: I'm thinking to leave this one alone for now as adding an example here seems redundant.

Regarding having an example for users that prefer one, our starter codes have test cases which I think serve that purpose better. Using test cases as examples will also work better in the future as our problems get more complicated to the point where they're impossible to format nicely in the problem description.

File: problems/lonelyNumber.yaml (hunk 3 above)
Comment (nicholashusin) [reply problems/lonelyNumber.yaml]: Fixed in the latest commit, thanks!

File: problems/lonelyNumber.yaml (hunk 4 above)
Comment (nicholashusin) [reply problems/lonelyNumber.yaml]: Fixed too!

File: problems/lonelyNumber.yaml (hunk 5 above)
Comment (nicholashusin) [reply problems/lonelyNumber.yaml]: Whoops, thanks for the catch! Fixed now.

File: problems/lonelyNumber.yaml (hunk 6 above)
Comment (nicholashusin) [reply problems/lonelyNumber.yaml:33]: Glad you like it :)

**Review by ifeomi — APPROVED** [review]

**Review by chronologos — APPROVED** [review]

**Pull request merged**